### PR TITLE
Remove implicit vars from store models

### PR DESCRIPTION
**Pull request by phlax**
more cleaning up unit.save

### CODE REVIEW

File: pootle/apps/pootle_store/models.py [hunk 1]
```diff
@@ -872,10 +872,7 @@ def toggle_qualitycheck(self, check_id, false_positive, user):
                          unit=self, store=self.store, type=sub_type,
                          quality_check=check)
         sub.save()
-
-        # update timestamp
-        # log user action
-        self.save()
+        self.save(revision=Revision.incr())
```
Comment (ta2-1) [inline pootle/apps/pootle_store/models.py]: @phlax why do we need to set new unit revision here?

Comment (phlax) [inline pootle/apps/pootle_store/models.py]: not 100% - but that was what i figured the effect of having those other vars set was

Comment (phlax) [inline pootle/apps/pootle_store/models.py]: @ta2-1 - looking at it tho - i think i just wanted to be sure that the revision was incremented - maybe we dont need to - maybe - its just stats that need a kick

**Comment by unho** [comment]
lgtm

File: pootle/apps/pootle_store/models.py (hunk 1 above)
Comment (unho) [inline pootle/apps/pootle_store/models.py]: Incrementing it is the safest choice.

**Comment by ta2-1** [comment]
lgtm

**Pull request merged**